chromium_src/chrome/browser/ui/webui/chrome_web_ui_configs.cc

```diff
@@ -33,9 +34,11 @@ void RegisterChromeWebUIConfigs() {
   map.AddWebUIConfig(std::make_unique<brave_rewards::RewardsPanelUIConfig>());
   map.AddWebUIConfig(std::make_unique<brave_rewards::TipPanelUIConfig>());
   map.AddWebUIConfig(std::make_unique<CookieListOptInUIConfig>());
+  map.AddWebUIConfig(std::make_unique<SpeedreaderToolbarUIConfig>());
```

Comment:
Nit: Should be one line below, if we're going for alphabetical order

Reply:
Done.